moveit_setup_assistant/README.md

```diff
@@ -1 +1,7 @@
 # MoveIt Setup Assistant
+
+## ROS2 port progress
+
+`ros2 run moveit_setup_assistant collisions_updater` - Works
+
+`ros2 run moveit_setup_assistant moveit_setup_assistant` - GUI starts, but crashes when loading URDF
```

Comment:
Drop this ( and the related work ) from this PR.  It is much better to not merge WIP and to have smaller PRs.